common/time/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
+
+pub mod time {
```

Comment:
why have a mod time here instead of top level?

Reply:
Yeah, I was thinking that maybe other stuff would come later in there. But I guess having a top level would be cleaner, I'll change it.